evm/its.js

```diff
@@ -614,6 +612,23 @@ async function processCommand(config, chain, options) {
                 }
             }
 
+            const contractNameFactory = 'InterchainTokenFactory';
+            const contractConfigFactory = chain.contracts[contractNameFactory];
+
+            const toCheckFactory = {
+                address: interchainTokenFactory,
+                implementation: interchainTokenFactoryImplementation,
+            };
+
+            for (const [key, value] of Object.entries(toCheckFactory)) {
```

Comment:
code duplication. refactor the comparison into a helper function, and call it twice

Reply:
added helper function